index.js

```diff
@@ -1,13 +1,12 @@
-/**
- * The JWT token you get after authenticating with our API.
- * Check the Authentication section of the documentation for more details.
- */
-const accessToken = "";
-const uniqueMeetingId = btoa("user@example.com");
-const symblEndpoint = `wss://api.symbl.ai/v1/realtime/insights/${uniqueMeetingId}?access_token=${accessToken}`;
-
+const accessToken = accessToken;
+// Refer to the Authentication section for how to generate the accessToken: https://docs.symbl.ai/docs/authenticate
+const uuid = require('uuid').v4;
```

Comment:
@symbldoc Is the code supposed to be run in a browser console? if yes this line might cause issues since this does not exist in the browser context

Reply:
I didn't get any error when I ran it in the browser, maybe it was just ignored? Should we remove the UUID lines here, or is there a better way?

Reply:
@symbldoc I suggest we write a basic function which creates a uuid, look at the function below

```js
function uuidv4() {
  return ([1e7]+-1e3+-4e3+-8e3+-1e11).replace(/[018]/g, c =>
    (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16)
  );
}

```
Source - https://stackoverflow.com/questions/105034/how-do-i-create-a-guid-uuid

Also, ChatGPT-generated this for me

```js
function generateUUID() {
  let dt = new Date().getTime();
  let uuid = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
    let r = (dt + Math.random()*16)%16 | 0;
    dt = Math.floor(dt/16);
    return (c=='x' ? r :(r&0x3|0x8)).toString(16);
  });
  return uuid;
}
```